exercises/01_sort4/sort4.go

```diff
@@ -3,7 +3,21 @@ package main
 import "fmt"
 
 func minmax(a, b int) (mn, mx int) {
-	return a, b
+	return min(a, b), max(a, b)
```

Comment:
Please consider simplifying this function:
```go
func minmax(a, b int) (mn, mx int) {
  if a <= b {
    return a, b
  }
  return b, a
}
```

No need to introduce unnecessary functions